libevm/pom.xml

```diff
@@ -3,7 +3,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>io.horizen</groupId>
   <artifactId>libevm</artifactId>
-  <version>1.1.0</version>
+  <version>1.1.0-snapshot1</version>
```

Comment:
new version must be 1.2.0-snapshot
(and please change the SDK accordingly)

Reply:
Done